### DISCUSSION TITLE
wrong dimension on PDF when converting to series of images

### DISCUSSION THREAD

**Discussion by sebawagner** [ImageMagick/ImageMagick#1891] [General] [answered]
Following PDF converts to images with whitespace. While the PDF itself doesn't show it. It seems like to assume the wrong width of the document.

Document: [73a45f3b-1b9c-44b9-96f4-cf9bbdb415cc.pdf](https://github.com/ImageMagick/ImageMagick/files/4467365/73a45f3b-1b9c-44b9-96f4-cf9bbdb415cc.pdf)

Generated image:
![page-0000](https://user-images.githubusercontent.com/5152064/79082551-036ac600-7d7b-11ea-9ea4-d46e3dd6cf05.png)

you can see a lot of whitespace on the left. As it has width of 4134px. It should be much less. Pretty much exactly 3 times less.

What is also strange is that on the 2nd page the whitespace is on the right:
![output-0001](https://user-images.githubusercontent.com/5152064/79082635-a91e3500-7d7b-11ea-931e-4e04c075a04a.png)

and so forth to the 3rd and 4th, and then it seems to start again on the left with whitespace. Like a pattern.

Command to generate PNGs
convert -density 150 73a45f3b-1b9c-44b9-96f4-cf9bbdb415cc.pdf +profile '*' -quality 90 output-%04d.png 

Versions:
```
convert -version
Version: ImageMagick 7.0.10-0 Q16 x86_64 2020-04-04 https://imagemagick.org
Copyright: © 1999-2020 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(3.1) 
Delegates (built-in): bzlib freetype gslib heic jng jp2 jpeg lcms ltdl lzma openexr png ps tiff webp xml zlib
```

```
gs -version
GPL Ghostscript 9.52 (2020-03-19)
Copyright (C) 2020 Artifex Software, Inc.  All rights reserved.
```

**Comment by urban-warrior** [comment, marked as answer]
Try this command:

`convert -density 150 -define pdf:use-cropbox=true 73a45f3b-1b9c-44b9-96f4-cf9bbdb415cc.pdf +profile '*' -quality 90 output-%04d.png`

**Reply by sebawagner** [reply]
Nice. That works fine. 
I wonder if there is any reason why you wouldn't use cropbox by default? 
Thanks
